src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -163,6 +166,13 @@ def __init__(self, config):
         # on DBSInterface anyway
         self.dbsUrl = self.config.DBS3Upload.dbsUrl
 
+        #Tier0 Agent don't need this
+        if hasattr(self.config, "Tier0Feeder"):
+            self.wmstatsServerSvc = None
+        else:
+            wmstatsSvcURL = (self.config.AgentStatusWatcher.centralWMStatsURL).replace("couchdb/wmstats", "wmstatsserver")
```

Comment:
Seangchan, it looks like this `centralWMStatsURL` attribute is defined for a few components. How about we set it under the General section just to avoid these cross-component attributes? That can be done in a different PR though.

Reply:
As we discussed we will do it in different PR. Since this has to be patched to current production agent.